blog/09-05-2023-smart-contract-security-on-celo-with-securify/smart-contract-security-on-celo-with-securify.md

```diff
@@ -0,0 +1,276 @@
+---
+title: Smart contract security on Celo with Securify
+description: Analyze smart contracts using free security tools - Slither
+authors:
+  - name: Isaac Jesse
+    title: Web3, Smart Contract Developer
+    url: https://github.com/bobeu
+    image_url: https://github.com/bobeu.png
+tags: [celosage, intermediate, smartcontract, solidity]
+hide_table_of_contents: true
+slug: /tutorials/smart-contract-security-on-celo-with-securify
+---
+
+![header](../../src/data-tutorials/showcase/intermediate/smart-contract-security-on-celo-with-securify.png)
+
+## Introduction
+
+Smart contracts are self-executing pieces of code deployed on a blockchain. EVM-compatible blockchains such as Celo are free from censorship, downtime, and third-party interference, and have since invented changed how software is perceived. However, as exciting as smart contracts are, they introduce known and probably undetected risks ranging from low to critical in severity. It is crucial for developers to thoroughly scrutinize contract code before deployment.
+
+## Prerequisites​
+This guide introduces you to working with a security tool called Slither. You must at least be an intermediate working with smart contracts using solidity. If you're new in this space, I recommend starting from **[here](https://docs.celo.org/blog/tutorials/building-a-solidity-smart-contract-for-nft-royalty-fees-a-step-by-step-guide)** and **[here](https://docs.celo.org/blog/tutorials/best-practices-for-writing-smart-contracts-with-real-world-examples)**.
+
+## Requirements​
+
+The following tools are recommended and should be installed before proceeding.
+
+- [Python](https://python.org)
+- Install an editor/IDE. VSCode preferably
+- Install NodeJs version 14 or later
+
+**What is Slither?**
+Slither is a static analysis framework for smart contracts development. With Slithet, developers are able to find vulnerabilities in smart contracts code written in solidity within seconds and optimize their code.
+
+Slither is a security tool for analyzing smart contracts. There are other smart contracts platforms such as TRON and so on. But this guide focuses on the Celo blockchain - `a platform acting as a global payment infrastructure for cryptocurrencies that aims to target mobile users, highly compatible with the Ethereum Virtual Machine. So if you write contracts that target the Celo platform, it is very important to prioritize security. This helps to detect and avoid a range of smart contract issues such as integer overflow/underflow, owner-overwrite-to-Ether-withdrawal, and many others.
+
+**Installing Slither**
+
+To use Slither, be sure Python is installed on your machine since we will need the Python package manager - Pip to install the tool. If you need to install it from scratch, please follow the steps I itemized **[here](https://docs.celo.org/blog/tutorials/interacting-with-smart-contract-on-celo-using-web3py)**. To install Slither, run the following commands.
+
+**Steps**
+
+1. 
```

Comment:
You can delete this section because it feels weird if it only has numbers without a title. It's best to show the block code that you want to show directly.